source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to synced collections, but they won't
+   be able to receive upload acknowledgements.
+
+#. **Building Flexible Sync metadata** 
+   
+   Clients cannot connect at this stage. The duration of this stage is directly
+   proportional to the amount of data in synced collections.
+
+#. **Completed migration** 
+
+   :guilabel:`Manage Sync Migration` appears on the Device Sync page if the 
+   migration hasn't been committed or reverted.
+
+After the migration has finished, check out the :ref:`After Migration
+<realm-sync-migrate-after>` section for more information about how
+Partition-Based Sync clients interact with the Flexible Sync backend.
+
+Migrate Partition-Based Sync to Flexible Sync
+---------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can revert it 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+Migration generally has similar effects as :ref:`terminating <terminating-realm-sync>` 
+and re-enabling Device Sync. There are some additional effects of migration:
+
+- In the App Services UI, the Rules and Schema pages will be read-only.
+- Your App will experience a period of read-only Sync and then a 
+  period of downtime.
+- The new Flexible Sync configuration will set 
+  :ref:`client max offline time <client-maximum-offline-time>` to 30 days.
+- Storage in your Atlas cluster used by Device Sync metadata is temporarily doubled.
+  Partition-Based Sync metadata and Flexible Sync metadata must temporarily co-exist. 
+  Depending on the size of your synced data, this could affect billing every time you
+  migrate.
+
+**To migrate from Partition-Based Sync to Flexible Sync:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select :guilabel:`[Migrate to Flexible Sync]`.
+   
+   .. image:: /images/begin-migration.png
+      :alt: Begin a migration
+   
+#. Read the information about migrating, then select 
+   :guilabel:`Next: Migration Requirements`.
+#. Ensure you meet the migration requirements, the select 
+   :guilabel:`Start Migration`.
+
+.. note:: Temporary Sync Disruption
+
+   Clients will not be able to sync their local writes until the migration has
+   completed.
+
+After a migration is completed, Partition-Based Sync client apps can communicate
```

Comment:
You mentioned above that clients must perform a client reset after migrating. Maybe it's worth adding that here?